frame/alliance/src/lib.rs

```diff
@@ -619,24 +648,80 @@ pub mod pallet {
 		}
 
 		/// Initialize the founders, fellows, and allies.
+		/// Founders must be provided to initialize the Alliance.
+		///
+		/// Provide witness data to disband current Alliance before initializing new.
+		/// Alliance must be empty or disband first to initialize new.
 		///
-		/// This should only be called once, and must be called by the Root origin.
-		#[pallet::weight(T::WeightInfo::init_members(
+		/// Alliance is only disband if new members set is not provided.
```

Comment:
I don't see this in the logic. It looks like it would `return Ok(())` if no new members were provided.
```suggestion
		/// Alliance is only disbanded if new member set is not provided.
```